assignments/function-conversion.js

```diff
@@ -1,21 +1,21 @@
 // Take the commented ES5 syntax and convert it to ES6 arrow Syntax
 
-// let myFunction = function () {
+// let myFunction = () => {
 // console.log("Function was invoked!");
 // };
 // myFunction();
 
-// let anotherFunction = function (param) {
+// let anotherFunction = (param) => {
 //   return param;
 // };
 // anotherFunction("Example");
```

Comment:
In a case of one parameter, curly brackets are optional but needed for clean code.